data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -395,42 +500,54 @@ def _workforce_factor(self) -> bool:
         # AND
         # Where the high school degree achievement rates for adults 25 years and older is less than 95%
         # (necessary to screen out university block groups)
-        workforce_criteria_for_states = (
-            (
-                self.df[
-                    field_names.UNEMPLOYMENT_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.MEDIAN_INCOME_PERCENT_AMI_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                # Note: a high median income as a % of AMI is good, so take 1 minus the threshold to invert it.
-                # and then look for median income lower than that (not greater than).
-                <= 1 - self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.POVERTY_LESS_THAN_100_FPL_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.LINGUISTIC_ISO_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-        )
-        workforce_combined_criteria_for_states = (
+
+        high_scool_achievement_rate_threshold = (
             self.df[field_names.HIGH_SCHOOL_ED_FIELD]
             >= self.LACK_OF_HIGH_SCHOOL_MINIMUM_THRESHOLD
-        ) & workforce_criteria_for_states
+        )
+
+        workforce_criteria_for_states_columns = [
+            field_names.UNEMPLOYMENT_FIELD
+            + field_names.PERCENTILE_FIELD_SUFFIX,
+            field_names.POVERTY_LESS_THAN_100_FPL_FIELD
+            + field_names.PERCENTILE_FIELD_SUFFIX,
+            field_names.LINGUISTIC_ISO_FIELD
+            + field_names.PERCENTILE_FIELD_SUFFIX,
+            field_names.MEDIAN_INCOME_PERCENT_AMI_FIELD
+            + field_names.PERCENTILE_FIELD_SUFFIX,
+        ]
+
+        # Workforce criteria for states fields that create indicator columns
+        # for each tract in order to indicate whether they met any of the four
+        # criteria. We will used this create individual indicator columns.
+        workforce_indicator_columns = [
+            field_names.UNEMPLOYMENT_LOW_HS_EDUCATION,
+            field_names.POVERTY_LOW_HS_EDUCATION,
+            field_names.LINGUISTIC_ISOLATION_LOW_HS_EDUCATION,
+            field_names.MEDIAN_INCOME_LOW_HS_EDUCATION,
+        ]
+
+        for index_, _ in enumerate(workforce_criteria_for_states_columns[:-1]):
```

Comment:
I think this line is a bit more complex than it needs to be. If you're taking this approach, you can just do: 

```
for index_ in range(workforce_criteria_for_states_columns - 1) 
```

Reply:
Also - to harp on my usual theme - why is this implemented this way here, whereas in the `_health_factor`, the same number of factors get handled individually? We should be consistent throughout and not take two different approaches to the same logic.

Reply:
Ok

Reply:
Just to be clear, we want `range(0, len(workforce_criteria_for_states_columns) - 1):`.